src/components/Auth/BackupRecoveryCodesView.tsx

```
@@ -0,0 +1,54 @@
+            </div>
+
+            <div className="flex gap-2">
+                <button
+                    type="button"
+                    onClick={handleCopy}
+                    className="flex-1 py-2.5 rounded-xl bg-slate-800 hover:bg-slate-700 text-xs font-bold text-slate-200 transition-all flex items-center justify-center gap-1.5"
+                >
+                    {copied ? <Check className="w-3.5 h-3.5 text-emerald-400" /> : <Copy className="w-3.5 h-3.5" />}
+                    <span>{copied ? 'Copied Codes!' : 'Copy Codes'}</span>
+                </button>
+                <button
```

ESLint

[error] Replace `················` with `········`

(prettier/prettier)